libsyclinterface/source/dpctl_sycl_device_interface.cpp

```diff
@@ -25,11 +25,11 @@
 //===----------------------------------------------------------------------===//
 
 #include "dpctl_sycl_device_interface.h"
-#include "../helper/include/dpctl_error_handlers.h"
-#include "../helper/include/dpctl_string_utils.hpp"
-#include "../helper/include/dpctl_utils_helper.h"
 #include "Support/CBindingWrapping.h"
+#include "dpctl_error_handlers.h"
+#include "dpctl_string_utils.hpp"
 #include "dpctl_sycl_device_manager.h"
+#include "dpctl_utils_helper.h"
 #include <CL/sycl.hpp> /* SYCL headers   */
```

Comment:
How about switching to `sycl.hpp` too?

Reply:
Tried both `<sycl/sycl.hpp>` and `<sycl.hpp>`. Both errored out. Going to merge as is.